src/toil/test/__init__.py

```diff
@@ -29,17 +28,13 @@
 from inspect import getsource
 from shutil import which
 from textwrap import dedent
+from unittest.util import strclass
+from urllib.request import urlopen
 
 import pytz
-from builtins import str
-from future.utils import with_metaclass
-from six import iteritems, itervalues
-from six.moves.urllib.request import urlopen
-from unittest.util import strclass
 
-from toil import subprocess
-from toil import toilPackageDirPath, applianceSelf, ApplianceImageNotFound
-from toil import which
+from toil import (ApplianceImageNotFound, applianceSelf, subprocess,
```

Comment:
Both `subprocess` and `which` here were only imported from toil to get py2/3 compatible imports, so they can now be:

```
import subprocess
from shutil import which
```